src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -37,134 +32,170 @@ type ValidatorNameProps = {
 }
 
 export const ValidatorName = ({ moniker, isStaking, validatorAddress }: ValidatorNameProps) => {
-  const isLoaded = true
   const assetIcon = isStaking
     ? `https://raw.githubusercontent.com/cosmostation/cosmostation_token_resource/master/moniker/cosmoshub/${validatorAddress}.png`
     : 'https://assets.coincap.io/assets/icons/256/atom.png'
 
   return (
     <Box cursor='pointer'>
       <Flex alignItems='center' maxWidth='180px' mr={'-20px'}>
-        <SkeletonCircle boxSize='8' isLoaded={isLoaded} mr={4}>
-          <AssetIcon src={assetIcon} boxSize='8' />
-        </SkeletonCircle>
-        <Skeleton isLoaded={isLoaded} cursor='pointer'>
-          {isStaking && (
-            <Tag colorScheme='blue'>
-              <TagLabel>{moniker}</TagLabel>
-            </Tag>
-          )}
-          {!isStaking && <RawText fontWeight='bold'>{`${moniker}`}</RawText>}
-        </Skeleton>
+        <AssetIcon src={assetIcon} boxSize='8' />
+        {isStaking ? (
+          <Tag colorScheme='blue'>
+            <TagLabel>{moniker}</TagLabel>
+          </Tag>
+        ) : (
+          <RawText fontWeight='bold'>{`${moniker}`}</RawText>
+        )}
       </Flex>
     </Box>
   )
 }
 
+const SHAPESHIFT_VALIDATOR_ADDRESS = 'cosmosvaloper199mlc7fr6ll5t54w7tts7f4s0cvnqgc59nmuxf'
+
 export const StakingOpportunities = ({ assetId }: StakingOpportunitiesProps) => {
   const asset = useAppSelector(state => selectAssetById(state, assetId))
   const marketData = useAppSelector(state => selectMarketDataById(state, assetId))
 
-  const { activeStakingOpportunities, stakingOpportunities, isLoaded } = useCosmosStakingBalances({
-    assetId,
-  })
+  const accountSpecifier = useAppSelector(state =>
+    selectFirstAccountSpecifierByChainId(state, asset?.chainId),
+  )
 
-  const hasActiveStakingOpportunities = activeStakingOpportunities.length !== 0
+  const stakingOpportunitiesData = useAppSelector(state =>
+    selectStakingOpportunitiesDataFull(state, { accountSpecifier, assetId }),
+  )
+
+  const hasActiveStaking =
+    // More than one opportunity data means we have more than the default opportunity
+    size(stakingOpportunitiesData) > 1 ||
+    // If there's only one staking but it isn't the default opportunity, then it's an active staking
+    stakingOpportunitiesData[0]?.address !== SHAPESHIFT_VALIDATOR_ADDRESS ||
+    bnOrZero(stakingOpportunitiesData[0]?.rewards).gt(0) ||
+    bnOrZero(stakingOpportunitiesData[0]?.totalDelegations).gt(0)
+
```

Comment:
Note: `totalDelegations` encapsulates `undelegatedEntries` + `delegatedAmount` - by checking for the absence of delegations/undelegations/rewards, we effectively get the guarantees we need that the opportunity is not in `active staking` state for the default ShapeShift validator.